streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1021,39 +1027,45 @@ public String toString() {
         final Sensor taskCreationSensor;
         final Sensor taskDestructionSensor;
 
-        public StreamsMetricsImpl(Metrics metrics) {
+        public StreamsMetricsImpl(Metrics metrics, String metricGrpName, String sensorNamePrefix,
```

Comment:
Fixed the variable name to streamsMetrics (was sensors)
 StreamsMetricsImpl is used 12 times. 

Also fixed the computation